### PR TITLE
Add postsubmit job for node-feature-discovery e2e testing

### PR DESCRIPTION
**Pull request by marquiz**

### CODE REVIEW

**Comment by marquiz** [comment]
Waiting for secret to be deployed
/hold

File: config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-postsubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      testgrid-tab-name: e2e-test-image
+      description: "Run end-to-end tests for node-feature-discovery after merging a pull request"
+    spec:
+      containers:
+      - image: golang:1.14
+        command:
+        - bash
+        args:
+        - c
+        - apt update && apt install -y jq && scripts/test-infra/test-e2e.sh
```
Comment (ArangoGutierrez) [inline config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-postsubmits.yaml:18]: isn't it a cleaner way of doing this?

Comment (marquiz) [reply config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-postsubmits.yaml:18]: The `apt` commands could be added to the `test-e2e.sh` script, but, I somehow find it cleaner just to hack them here in the job config.

Comment (marquiz) [reply config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-postsubmits.yaml:18]: In any case, I tried to keep the test script platform/OS agnostic and as simple as possible.

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Comment by marquiz** [comment]
Secret now deployed
/unhold

**Pull request merged**